### ISSUE TITLE
Mobilenet parameters: alpha & rho

### ISSUE THREAD

**Issue by joyccino** [tensorflow/tensorflow#46947]
Hello, my name is Joy and I've been learning about PoseNet in order to use it in my health-related research work.
I was impressed how mobilenet enables to keep high accuracy while reducing CPU (or GPU/NPU) dependency by adapting few parameters where my questions sprouted.
I've noticed that in mobilenet official papers, there were two multipliers introduced: alpha and rho. I'll skip the explanation of both parameters. 
I wonder what is the each value of alpha and rho for the mobilenet for the newest pretrained PoseNet model. Also, I'm wondering if there is a guideline for parameters(especially alpha and rho) tuning, and how the values of both are set and validated before training the model.
Thank you.

**Comment by Saduf2019** [comment]
@joyccino 
The multipliers determine  how depths are adjusted in positive Convs, generally more the depths better it is but that comes along with an expense of increased number of parameters.
As this is not a bug or a feature request, can you please create an issue on [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) for this and move this to closed status.
